File: Materal.APP/src/Deploy/Deploy.DataTransmitModel/ApplicationInfo/ApplicationInfoDTO.cs
using Deploy.Enums;
using System;

namespace Deploy.DataTransmitModel.ApplicationInfo
{
    /// <summary>
    /// 应用程序信息数据传输模型
    /// </summary>
    public class ApplicationInfoDTO
    {
        /// <summary>
        /// 唯一标识
        /// </summary>
        public Guid ID { get; set; }
        /// <summary>
        /// 名称
        /// </summary>
        public string Name { get; set; }
        /// <summary>
        /// 路径
        /// </summary>
        public string Path { get; set; }
        /// <summary>
        /// 主模块
        /// </summary>
        public string MainModule { get; set; }
        /// <summary>
        /// 类型
        /// </summary>
        public ApplicationTypeEnum ApplicationType { get; set; }
        /// <summary>
        /// 运行参数
        /// </summary>
        public string RunParams { get; set; }
    }
}

File: Level5/RC/RC.ServerCenter/RC.ServerCenter.Web/src/models/applicationInfo/ApplicationInfoDTO.ts
export default interface ApplicationInfoDTO {
    /**
     * 唯一标识
     */
    ID: string;
    /**
     * 名称
     */
    Name: string;
    /**
     * 路径
     */
    RootPath: string;
    /**
     * 主模块
     */
    MainModule: string;
    /**
     * 应用类型
     */
    ApplicationType: number;
    /**
     * 应用类型文本
     */
    ApplicationTypeTxt: string;
    /**
     * 是增量更新
     */
    IsIncrementalUpdating: boolean;
    /**
     * 运行参数
     */
    RunParams: string;
    /**
     * 应用状态
     */
    ApplicationStatus: number;
    /**
     * 应用状态文本
     */
    ApplicationStatusTxt: string;
}